game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/HelpTexts.java

```diff
@@ -0,0 +1,135 @@
+package games.strategy.engine.framework.startup.ui.posted.game;
+
+import lombok.experimental.UtilityClass;
+
+/**
+ * Stores help text constants and helper methods to create help texts used in the PBF and PBEM
+ * screens.
+ */
+@UtilityClass
+public class HelpTexts {
+
+  public static final String TRIPLEA_FORUM =
+      "<html><p style='width: 400px;'>"
+          + "Posts to forums.triplea-game.org<br/>"
+          + "You can play PBEM/PBF games via forums.triplea-game.org.<br/>"
+          + "Instructions:<br/>"
+          + "Create a new Forum post in the Play by Forum category "
+          + "https://forums.triplea-game.org/category/6/<br/>"
+          + "Copy the topic id from the URL displayed in the address bar of you "
+          + "browser. "
+          + "If the URL is https://forums.triplea-game.org/topic/24/ put the topic "
+          + "number (24) into the Topic ID field in TripleA<br/>"
+          + "Put your username and password for the forums.triplea-game.org forum "
+          + "into the username and password fields<br/>"
+          + "Click the Test Post button, to check that TripleA can reply to your "
+          + "forum post<br/>"
+          + "<i>Note:</i> Your forums.triplea-game.org username and password are not stored "
+          + "as part of the save game, but they are stored encrypted in the local "
+          + "file system if you select the option to remember your credentials. You "
+          + "may have to enter your username and password again if you open the save "
+          + "game on another computer."
+          + "</p></html>";
+  public static final String AXIS_AND_ALLIES_DOT_ORG_FORUM =
+      "<html><p style='width: 400px;'>"
+          + "Posts to www.AxisAndAllies.org<br/>"
+          + "This poster is build for PBEM games via AxisAndAllies.org. Instructions:"
+          + "<br/>"
+          + "Create a new Forum post in the Play Boardgames section "
+          + "http://www.axisandallies.org/forums/index.php?board=40.0<br/>"
+          + "Copy the topic id from the URL displayed in the address bar of you "
+          + "browser. If the URL is "
+          + "http://www.axisandallies.org/forums/index.php?topic=25252.0, "
+          + "put this 5 digit number (25252) into the Topic ID field in TripleA<br/>"
+          + "Put your username and password for the AxisAndAllies.org forum into the "
+          + "username and password fields<br/>"
+          + "Click the Test Post button, to check that TripleA can reply to your forum "
+          + "post<br/>"
+          + "<i>Note:</i> Your AxisAndAllies.org username and password are not stored as part "
+          + "of the save game, but they are stored encrypted in the local file system "
+          + "if you select the option to remember your credentials. You may have to "
+          + "enter your username and password again if you open the save game on "
+          + "another computer."
+          + "</p></html>";
```

Comment:
This is no longer accurate, the axis & allies forum migrated to NodeBB quite some time ago, so the description should be more or less equivalent to our forum

Reply:
The help text could use improvement, it has not been given much love, just mainly restored so far. No PR so far has really sought to change or improve it.